dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/formatting2/XtxtUMLFormatter.xtend

```diff
@@ -85,6 +87,16 @@ class XtxtUMLFormatter extends XbaseFormatter {
 		regionFor.keyword('extends').surround[oneSpace];
 	}
 
+	def dispatch void format(TUEnumeration it, extension IFormattableDocument document) {
```

Comment:
On the first hand, if semicolons are optional at the end of items, why make commas mandatory?

On the other hand, making them mandatory would simplify nothing as they would still not be part of the `TUEnumerationLiteral` parser rule and therefore the default block element formatter would put the new lines between the literal name and the comma.